src/containers/main/Dashboard/components/BlockExplorerMini/BlockScrollList/BlockScrollList.tsx

```
@@ -0,0 +1,109 @@
+                <Suspense fallback={<div></div>}>
+                    {data &&
+                        data.map(({ id, minersSolved, reward, timeAgo, blocks }) => (
+                            <BlockEntry
+                                key={id}
+                                id={id}
+                                minersSolved={minersSolved}
+                                reward={reward}
+                                timeAgo={timeAgo}
+                                blocks={blocks}
+                            />
+                        ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)